vss/README.md

```diff
@@ -55,12 +55,12 @@ use the full name. When official release is created replace the copied *.json-fi
 ## Tests after update
 
 ### Kuksa-val-server unit tests
-* Run kuksa-val-server unit tests according to [documentation](https://github.com/eclipse/kuksa.val/kuksa-val-server/test/unit-test/readme.md)
+* Run kuksa-val-server unit tests according to [documentation](https://github.com/eclipse-archived/kuksa.val/blob/master/kuksa-val-server/test/unit-test/readme.md)
 
 ### Kuksa-val-server smoke test
-* Build and start kuksa-val-server with new VSS release as described in the [README](https://github.com/eclipse/kuksa.val/blob/master/kuksa-val-server/README.md)
-* If needed [generate new certificates](https://github.com/eclipse/kuksa.val/tree/master/kuksa_certificates)
-* [Start Kuksa Client](https://github.com/eclipse/kuksa.val/blob/master/kuksa-client/README.md) and perform some basic tests that VSS changes are present
+* Build and start kuksa-val-server with new VSS release as described in the [README](https://github.com/eclipse-archived/kuksa.val/blob/master/kuksa-val-server/README.md)
+* If needed [generate new certificates](https://github.com/eclipse-kuksa/kuksa-databroker/tree/main/certificates)
```

Comment:
Is this the better link? https://github.com/eclipse-kuksa/kuksa-common/tree/main/tls ?